.github/workflows/ci.yml

```diff
@@ -123,12 +123,12 @@ jobs:
           if [[ "${{ matrix.test-path }}" == "tests/distributed/test_against_external_daemon/" ]]; then
             docker build -f Dockerfiles/debianx.Dockerfile -t jinaai/jina:test-daemon .
             docker run --name jinad -v /var/run/docker.sock:/var/run/docker.sock --network=host -d jinaai/jina:test-daemon
-            pytest --suppress-no-test-exit-code --force-flaky --min-passes 1 --max-runs 5 --cov=jina --cov-report=xml -n 1 --timeout=120 -v --ignore-glob='tests/integration/hub_usage/dummyhub*' ${{ matrix.test-path }}
+            pytest -s --suppress-no-test-exit-code --force-flaky --min-passes 1 --max-runs 5 --cov=jina --cov-report=xml -n 1 --timeout=180 -v --ignore-glob='tests/integration/hub_usage/dummyhub*' ${{ matrix.test-path }}
             docker rm -f jinad
           else
-            pytest --suppress-no-test-exit-code --force-flaky --min-passes 1 --max-runs 5 --cov=jina --cov-report=xml -n 1 --timeout=120 -v --ignore-glob='tests/integration/hub_usage/dummyhub*' ${{ matrix.test-path }}
+            pytest -s --suppress-no-test-exit-code --force-flaky --min-passes 1 --max-runs 5 --cov=jina --cov-report=xml -n 1 --timeout=180 -v --ignore-glob='tests/integration/hub_usage/dummyhub*' ${{ matrix.test-path }}
           fi
-        timeout-minutes: 15
+        timeout-minutes: 20
```

Comment:
What's the difference between this one and the one from pytest? We should bump it up

Reply:
This is the timeout for each Test in jobs. 
![image](https://user-images.githubusercontent.com/44033547/108509748-a3428b80-728b-11eb-9bdf-6536e56debfd.png)

Reply:
It is the global timeout I think

Reply:
Oh probably, since we're running parallely